KALTURAPlayerSDK/GoogleCastProvider.m

```diff
@@ -214,7 +214,6 @@ - (GCKCastSession *)currentSession {
 
 - (void)setMediaSrc:(NSString *)mediaSrc {
     if (_mediaSrc != nil) {
-        _isChangeMedia = YES;
```

Comment:
looks like you dont need here if & else please make sure to merge it to one line